.circleci/config.yml

```diff
@@ -619,8 +619,6 @@ jobs:
             kubectl -n ci-eugene expose pod elasticsearch --port 9200 --name=elasticsearch
             # Deploy Kibana
             bin/arnold -c eugene -e ci -a kibana deploy
-            # Test service deployed with the next route
-            bin/ci route -s kibana -c green -p "/api/status" -P next
```

Comment:
This endpoint does not exists any more?

Reply:
No it does, but the configuration is not blue-green compatible (see the `server.publicBaseUrl` in the application ConfigMap). We only test the deployment after having switched it.